website/source/docs/builders/ncloud.html.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
these docs need a little bit of work.

This page should be added to source/layouts/docs.erb

and you should test this by navigating to `website/` and running `make website. (make sure docker is installed).

You can then view this page at http://localhost:4567/docs/builders/ncloud.html

It should look similar to the existing builder pages.

Reply:
Ok, I'll try it. Thank you.